erclient/client.py

```diff
@@ -817,6 +817,16 @@ def get_subject_tracks(self, subject_id='', start=None, end=None):
 
         return self._get(path='subject/{0}/tracks'.format(subject_id), params=p)
 
+    def get_subject_source_tracks(self, subject_id='', src_id='', start=None):
+        """
+        Get the latest tracks for the Subject having the given subject_id and a source ID.
+        """
+        p = {}
+        if start is not None and isinstance(start, datetime):
+            p['since'] = start.isoformat()
+
```

Comment:
Looks like when the start parameter isn't a datetime then it'll be ignored silently. Then the caller may get a different result back but he won't notice the error.
Maybe we can do something like:
```suggestion
        if start:
            assert isinstance(start, datetime)
            p['since'] = start.isoformat()
```